draft-ietf-ppm-dap.md

```diff
@@ -1715,6 +1718,33 @@ following checks:
 
 If all of the above checks succeed, the input share is not marked as invalid.
 
+#### Report ID ranges {#report-id-ranges}
+
+Replay checking is implemented using "report ID ranges". A report ID range is an
+interval of report IDs (using lexicographic ordering), along with an associated
+timestamp.
+
+A set of reports for a task determines a set of report ID ranges. The report ID
+ranges associated with a given set of reports are determined by the following
+procedure:
+
+1. Round each report's timestamp down to the nearest multiple of
+   `time_precision`, and partition the set of reports into groups based on this
+   rounded timestamp.
```

Comment:
Let's make sure this is harmonized with the rest of the draft. I believe we say "truncate", not "round". However I haven't checked.

Reply:
We actually use both! I prefer `truncate` so I switched to that.